docs/DeveloperGuide.md

```diff
@@ -283,32 +283,165 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `BobTheBodyBuilder` and the **Actor** is the `Gym Manager`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case:  Add a client**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  Gym Manager chooses to add a client
+2.  Gym Manager enters details in required format
+3.  BobTheBodyBuilder displays new client 
```

Comment:
As mentioned above, can you change all the instances of BobTheBodyBuilder to BBTB instead
```suggestion
3.  BBTB displays new client 
```